<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <item>
     <widget class="QScrollArea" name="optionsScroll">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="optionsColumn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="optionGlobal">
          <property name="title">
           <string>Global</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="fpsL">
             <property name="text">
              <string>FPS</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundSpinBox" name="fps">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="amplificationL">
             <property name="text">
              <string>Amplification</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="BoundDoubleSpinBox" name="amplification">
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="trigger_msL">
             <property name="text">
              <string>Trigger Width (ms)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="BoundSpinBox" name="trigger_ms">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="render_msL">
             <property name="text">
              <string>Render Width (ms)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="BoundSpinBox" name="render_ms">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="optionChannel">
          <property name="title">
           <string>Performance</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="render_subfpsL">
             <property name="text">
              <string>Render FPS Divisor</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundSpinBox" name="render_subfps">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="trigger_subsamplingL">
             <property name="text">
              <string>Trigger Subsampling</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="BoundSpinBox" name="trigger_subsampling">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="render_subsamplingL">
             <property name="text">
              <string>Render Subsampling</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="BoundSpinBox" name="render_subsampling">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="optionAppear">
          <property name="title">
           <string>Appearance</string>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="render__bg_colorL">
             <property name="text">
              <string>Background</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundLineEdit" name="render__bg_color">
             <property name="text">
              <string>bg</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="render__init_line_colorL">
             <property name="text">
              <string>Line Color</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="BoundLineEdit" name="render__init_line_color">
             <property name="text">
              <string>fg</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="render__line_widthL">
             <property name="text">
              <string>Line Width</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="BoundDoubleSpinBox" name="render__line_width">
             <property name="minimum">
              <double>0.500000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="optionLayout">
          <property name="title">
           <string>Layout</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="layout__orientationL">
             <property name="text">
              <string>Orientation</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="BoundComboBox" name="layout__orientation"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="layout__ncolsL">
             <property name="text">
              <string>Columns</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="layoutDims">
             <item>
              <widget class="BoundSpinBox" name="layout__ncols">
               <property name="specialValueText">
                <string notr="true"> </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="layout__nrowsL">
               <property name="text">
                <string>Rows</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="BoundSpinBox" name="layout__nrows">
               <property name="specialValueText">
                <string notr="true"> </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="render_video_sizeL">
             <property name="text">
              <string>Video Size</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="BoundLineEdit" name="render_video_size">
             <property name="text">
              <string>vs</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="audioColumn">
      <item>
       <widget class="QGroupBox" name="audioGroup">
        <property name="title">
         <string>Master Audio</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="BoundLineEdit" name="master_audio">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="master_audio_browse">
           <property name="text">
            <string>&amp;Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="channelsGroup">
        <property name="title">
         <string>Oscilloscope Channels</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="channelBar">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="ShortcutButton" name="channelAdd">
             <property name="text">
              <string>&amp;Add...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ShortcutButton" name="channelDelete">
             <property name="text">
              <string>&amp;Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ShortcutButton" name="channelUp">
             <property name="text">
              <string>Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ShortcutButton" name="channelDown">
             <property name="text">
              <string>Down</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="ChannelTableView" name="channel_view"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPlay"/>
    <addaction name="actionRender"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionRender"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>&amp;Play</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRender">
   <property name="text">
    <string>&amp;Render to Video</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BoundLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ovgenpy/gui/data_bind.h</header>
  </customwidget>
  <customwidget>
   <class>BoundSpinBox</class>
   <extends>QSpinBox</extends>
   <header>ovgenpy/gui/data_bind.h</header>
  </customwidget>
  <customwidget>
   <class>BoundDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ovgenpy/gui/data_bind.h</header>
  </customwidget>
  <customwidget>
   <class>BoundComboBox</class>
   <extends>QComboBox</extends>
   <header>ovgenpy/gui/data_bind.h</header>
  </customwidget>
  <customwidget>
   <class>ShortcutButton</class>
   <extends>QPushButton</extends>
   <header>ovgenpy/gui/__init__.h</header>
  </customwidget>
  <customwidget>
   <class>ChannelTableView</class>
   <extends>QTableView</extends>
   <header>ovgenpy/gui/__init__.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
